pkg/lsp/service.go

```diff
@@ -62,6 +62,11 @@ type ServerCapabilities struct {
 	// XDefinitionProvider indicates the server provides support for
 	// textDocument/xdefinition. This is a Sourcegraph extension.
 	XDefinitionProvider bool `json:"xdefinitionProvider,omitempty"`
+
+	// XWorkspaceSymbolByProperties indicates the server provides support for
+	// querying symbols by properties with WorkspaceSymbolParams.symbol. This
+	// is a Sourcegraph extension.
+	XWorkspaceSymbolByProperties bool `json:"xworkspaceSymbolByProperties,omitempty"`
```

Comment:
@beyang because that would imply that `workspace/symbol` is a new method. What's new is only the ability to query by symbol properties.